coderd/coderd.go

```diff
@@ -1007,6 +1007,12 @@ func New(options *Options) *API {
 						})
 					})
 				})
+				r.Route("/provisionerdaemons", func(r chi.Router) {
+					r.Get("/", api.provisionerDaemons)
+				})
```

Comment:
If we add filtering by `key`, can we merge this with `/organizations/{organization}/provisionerkeys/daemons`? Or at least merge the code they both invoke? If possible, just to make sure we are consistent.

https://github.com/coder/coder/blob/dc701145c39805617b6f513b29b95339ea553658/enterprise/coderd/provisionerkeys.go#L129-L129

Reply:
That's a good idea, I wasn't sure how much to move to AGPL vs enterprise but none of this code is critical to keep in enterprise 👍🏻

Reply:
Generally I think `GET` endpoints can be made AGPL, and `POST`/`PATCH` should be enterprise.